packages/runner-ct/src/app/app.jsx

```diff
@@ -1,169 +1,61 @@
-import cs from 'classnames'
-import { observable } from 'mobx'
 import { observer } from 'mobx-react'
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
-import { findDOMNode } from 'react-dom'
+import React from 'react'
 import { Reporter } from '@packages/reporter'
-import { $ } from '@packages/driver'
 
 import errorMessages from '../errors/error-messages'
 import util from '../lib/util'
 import State from '../lib/state'
 
 import SpecsList from '../specs/specs-list'
+import SplitPane from 'react-split-pane'
```

Comment:
not sure if we have a standard around imports but seems the standard in this file is `node_modules` at the top. but this is not a blocker, just something that I noticed.